.NET/Microsoft.Recognizers.Text.DateTime/Utilities/MergedParserUtil.cs

```diff
@@ -520,6 +520,39 @@ public static List<DateTimeParseResult> DateTimeResolutionForSplit(DateTimeParse
             return results;
         }
 
+        public static DateTimeParseResult TasksModeModification(DateTimeParseResult slot, DateObject referenceTime)
+        {
+            switch (slot.Type.Substring(ParserTypeName.Length + 1))
+            {
+                case Constants.SYS_DATETIME_DATE:
+                    slot = ModifyDate(slot, referenceTime);
```

Comment:
All these new methods only called from here need better names to illustrate what they are doing and that they "belong" to the new mode. Perhaps also move them to their own static file like TasksModeProcessing.

Please add a code comment on each too.

Reply:
Added TasksModeProcessing file for handling DateTime modification under TasksMode and changed named accordingly.